react/features/base/react/components/native/styles.js

```diff
@@ -8,8 +8,8 @@ const AVATAR_OPACITY = 0.4;
 const AVATAR_SIZE = 65;
 const HEADER_COLOR = ColorPalette.blue;
 
-// Header height is from iOS guidelines. Also, this looks good.
-const HEADER_HEIGHT = 44;
+// Header height is from Android guidelines. Also, this looks good.
+const HEADER_HEIGHT = 56;
```

Comment:
why are we not using each platform's default here?

Reply:
Because 44 is reported to be too small in iOS too